control/latlon_muxer/include/latlon_muxer/node.hpp

```diff
@@ -1,46 +1,44 @@
-/*
- * Copyright 2015-2019 Autoware Foundation. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#ifndef CONTROL_LATLON_MUXER_INCLUDE_LATLON_MUXER_NODE_HPP_
-#define CONTROL_LATLON_MUXER_INCLUDE_LATLON_MUXER_NODE_HPP_
+// Copyright 2015-2020 Autoware Foundation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LATLON_MUXER__NODE_HPP_
```

Comment:
Is the include guard added by e.g. clang-format? Or did you change it manually? The other format seemed a bit safer. Btw, was there a decision to not use `#pragma once`?

Reply:
That's part of the ROS 2 linters (uncrustify or cppcheck, can't recall now). Anyway, it's in the ROS 2 style guide, which in turn is based on Google's:

https://google.github.io/styleguide/cppguide.html#The__define_Guard

Reply:
I have been in involved in way to long discussions about this topic so  I know too many pros and cons. Just wanted to know if and what the policy and desired format is here. Thanks for the reference to the ROS2 style guide so we have a clear guideline @esteve